src/Silex/Provider/ValidatorServiceProvider.php

```diff
@@ -46,7 +48,21 @@ public function register(Container $app)
         };
 
         $app['validator.mapping.class_metadata_factory'] = function ($app) {
-            return new LazyLoadingMetadataFactory(new StaticMethodLoader());
+            $loader = null;
```

Comment:
Ok, I've added a test case for the setup and updated the docs.
The builds seems to be failing, but as I have seen, this is caused for using a PHPUnit 6+ using the the old `\PHPUnit_Framework_TestCase` syntax.